lib/order.rb

```diff
@@ -0,0 +1,17 @@
+
+class Order
+
+   def initialize
+    @total_price = 0
+    
+   end
+
+   def sum(food_list)
+    food_list.each { |hash| hash.each {|food, price| @total_price += price } }
+    @total_price
```

Comment:
`total_price` should not be an instance variable here. A local variable is enough :)The user calls the sum method to get that price.